### PR TITLE
* Update to latest calculations for moveset ratings

### PR DESCRIPTION
**Pull request by 607ch00**
https://www.reddit.com/r/TheSilphRoad/comments/4vcobt/posthotfix_pokemon_go_full_moveset_rankings/d6gun8y
- Added three additional columns for moveset ratings including IV of a particular pokemon

### CODE REVIEW

File: src/me/corriekay/pokegoutil/windows/PokemonTab.java [hunk 1]
```diff
@@ -850,10 +862,13 @@ private void constructNewTableModel(PokemonGo go, List<Pokemon> pokes) {
                 caughtCol = new ArrayList<>(),//22
                 favCol = new ArrayList<>();//23
         private final ArrayList<Long> duelAbilityCol = new ArrayList<>();//24
-        private final ArrayList<Long> gymOffenseCol = new ArrayList<>();//25
+        private final ArrayList<Double> gymOffenseCol = new ArrayList<>();//25
```
Comment (Wolfsblvt) [inline src/me/corriekay/pokegoutil/windows/PokemonTab.java:891]: Why does gymOffense has to be a Double? Thought it is a number without decomals too?

**Comment by Wolfsblvt** [comment]
Sadly the other branches got merged first and I forgot that I've added another column. Guess you have to merge the latest stuff to your branch and resolve the problems manually, like increasing the ID numbers :-/

File: src/me/corriekay/pokegoutil/windows/PokemonTab.java (hunk 1 above)
Comment (607ch00) [inline src/me/corriekay/pokegoutil/windows/PokemonTab.java:891]: gymOffense potentially has a .5 attached, it was throwing off other values occasionally on me before, took me forever to figure out why some pokemons were just slightly off.

**Comment by 607ch00** [comment]
Hmmm.. still has conflicts.. why! !!

**Comment by 607ch00** [comment]
Finally conflict free :+1: I had to update my upstream remote...

**Comment by Wolfsblvt** [comment]
Tested, and works.

**Pull request merged**